rcl_logging_spdlog/src/rcl_logging_spdlog.cpp

```diff
@@ -53,6 +55,42 @@ static spdlog::level::level_enum map_external_log_level_to_library_level(int ext
   return level;
 }
 
+namespace
+{
+
+RCL_LOGGING_INTERFACE_LOCAL
+bool
+get_should_use_old_flushing_behavior()
+{
+  const char * env_var_name = "RCL_LOGGING_SPDLOG_EXPERIMENTAL_OLD_FLUSHING_BEHAVIOR";
```

Comment:
I think any of `RCL_LOGGING_SPDLOG_OLD_FLUSHING_BEHAVIOR`, `RCL_SPDLOG_OLD_FLUSHING_BEHAVIOR` or `RCL_LOGGING_OLD_FLUSHING_BEHAVIOR` would be shorter and still clear enough.

Reply:
Ok, `RCL_LOGGING_SPDLOG_OLD_FLUSHING_BEHAVIOR` is my favorite of those.

The `EXPERIMENTAL` part was a nod to the fact that we want this to be temporary (remove it later without notice). I also think the full `RCL_LOGGING_SPDLOG` prefix is good because it matches the package name, again indicating it's very narrow (as opposed to a full `ROS_` env var) and therefore subject to removal.

Anyone else have feelings on it?

Reply:
Ok, since I haven't heard from others, I'm going to leave it as-is for the reasons I stated above.